lib/verify.js

```diff
@@ -57,14 +57,6 @@ export async function barcodeToEnvelopedCredential({text, barcode} = {}) {
   return {credential};
 }
 
-export function isExpired({credential, now = new Date(), maxClockSkew}) {
-  if(credential.validUntil === undefined) {
-    return false;
-  }
-  const validUntil = new Date(credential.validUntil);
-  return _compareTime({t1: now, t2: validUntil, maxClockSkew}) >= 0;
-}
-
```

Comment:
Let's keep the coordinator checks expiry feature in case we need to use workflows that don't check it -- such that we can use the same workflow for cases where we check / don't check in the same coordinator. So, let's just add that code that includes `result.credential` when there's an error in this PR and not change anything else.